lib/tasks/assignments.rake

```diff
@@ -71,7 +98,7 @@ namespace :db do
         group_name_autogenerated: true,
         group_name_displayed: false,
         repository_folder: 'A3',
-        due_date: 2.months.from_now,
+        due_date: 1.months.from_now,
```

Comment:
I'm not sure why you changed this or the other due dates for these assignments; I'd prefer if you revert these changes and keep your changes localized to the new assignment.